### PR TITLE
[microNPU][ETHOSU] Fix SoftMax legalization parameters

### PR DESCRIPTION
**Pull request by Aleksei-grovety**
Fix SoftMax legalization parameters according to the values in Vela.

cc @neildhickey, @lhutton1, @ekalda, @leandron

### CODE REVIEW

**Review by ekalda — COMMENTED** [review]
Thanks for the PR @Aleksei-grovety! I'm wondering if there could be a test included that exercises this change?

**Comment by Aleksei-grovety** [comment]
I checked the output of the generated commands in TVM and Vela and updated legalization test accordingly.

File: tests/python/contrib/test_ethosu/test_legalize.py [hunk 1]
```diff
@@ -3526,20 +3528,72 @@ def representative_dataset():
...
...dxtN5kob+IbeXBgmc1xpB0mc9y0gnRca+N1nk1Idz5osXj9xx56oXmQe3KKBn19B4F+////fw==",     "P6G0lvBAXt0AAAAAAAAAAAEAAAAAAAAABAAAAAAgAQABAAAAAAAAAAEAAAAAAAAAAQAAAAAAAAABAAAAAAAAAAQAAAAAAAAADAAAAA==",     "P6G0lvBAXt0AAAAAAAAAAAEAAAAAAAAABAAAAAAgAQABAAAAAAAAAAEAAAAAAAAAAQAAAAAAAAABAAAAAAAAAAQAAAAAAAAAAQAAAA==",     "P6G0lvBAXt0AAAAAAAAAAAEAAAAAAAAABAAAAAAgAQABAAAAAAAAAAEAAAAAAAAAAQAAAAAAAAABAAAAAAAAAAQAAAAAAAAAAAAAQA==",     "P6G0lvBAXt0AAAAAAAAAAAEAAAAAAAAABAAAAAAgAQABAAAAAAAAAAEAAAAAAAAAAQAAAAAAAAABAAAAAAAAAAQAAAAAAAAA////fw==",     "P6G0lvBAXt0AAAAAAAAAAAEAAAAAAAAABAAAAAAgAQABAAAAAAAAAAEAAAAAAAAAAQAAAAAAAAABAAAAAAAAAAQAAAAAAAAAxMPDww==",     "P6G0lvBAXt0AAAAAAAAAAAEAAAAAAAAABAAAAAAgAQABAAAAAAAAAAEAAAAAAAAAAQAAAAAAAAABAAAAAAAAAAQAAAAAAAAAWlpaWg==",     "P6G0lvBAXt0AAAAAAAAAAAEAAAAAAAAABAAAAAAgAQABAAAAAAAAAAEAAAAAAAAAAQAAAAAAAAABAAAAAAAAAAQAAAAAAAAAAAAAIA==",     "P6G0lvBAXt0AAAAAAAAAAAEAAAAAAAAABAAAAAAgAQABAAAAAAAAAAEAAAAAAAAAAQAAAAAAAAABAAAAAAAAAAQAAAAAAAAABAAAAA==",     "P6G0lvBAXt0AAAAAAAAAAAEAAAAAAAAABAAAAAAgAQABAAAAAAAAAAEAAAAAAAAAAQAAAAAAAAABAAAAAAAAAAQAAAAAAAAAAgAAAA==",     "P6G0lvBAXt0AAAAAAAAAAAEAAAAAAAAABAAAAAAgAQABAAAAAAAAAAEAAAAAAAAAAQAAAAAAAAABAAAAAAAAAAQAAAAAAAAAIwAAAA=="  ]}
```
[the remark this answers is not in the document]
Comment (Aleksei-grovety) [reply tests/python/contrib/test_ethosu/test_legalize.py:3581]: I discovered the clipping problem when I started investigating the mismatch of the results on the hardware after the legalization of SoftMax operation. After fixing the problem with minimum size of a copied buffer, the results of networks with SoftMax operation on hardware and FVP coincide with the inference even without fixes from this PR.
I'll correct the test to check the attributes as we visit the graph.

**Comment by lhutton1** [comment]
@tvm-bot rerun

**Comment by lhutton1** [comment]
Thanks @Aleksei-grovety @ekalda!

**Pull request merged**